From aec3b64b0aaea0178b44ea65a204d76696e6a45e Mon Sep 17 00:00:00 2001
From: Leon Anavi <leon.anavi@konsulko.com>
Date: Mon, 10 Jan 2022 14:04:28 +0200
Subject: [PATCH] setup.py: Do not strip debugging symbols

Do not strip debugging symbols from Linux binaries to avoid
bitbake build errors related to already-stripped.

Signed-off-by: Leon Anavi <leon.anavi@konsulko.com>
---
Upstream-Status: Pending

 setup.py | 8 +-------
 1 file changed, 1 insertion(+), 7 deletions(-)

diff --git a/setup.py b/setup.py
index b98963b..d42f07a 100644
--- a/setup.py
+++ b/setup.py
@@ -19,13 +19,7 @@ if not dconv_libs:
     dconv_source_files.extend(glob("./deps/double-conversion/double-conversion/*.cc"))
 dconv_source_files.append("./lib/dconv_wrapper.cc")
 
-if platform.system() == "Linux" and environ.get("UJSON_BUILD_NO_STRIP", "0") not in (
-    "1",
-    "True",
-):
-    strip_flags = ["-Wl,--strip-all"]
-else:
-    strip_flags = []
+strip_flags = []
 
 module1 = Extension(
     "ujson",
-- 
2.25.1

